packages/amplify_datastore/lib/method_channel_datastore.dart

```diff
@@ -66,14 +69,16 @@ class AmplifyDataStoreMethodChannel extends AmplifyDataStore {
       QueryPagination? pagination,
       List<QuerySortBy>? sortBy}) async {
     try {
-      final List<Map<dynamic, dynamic>> serializedResults =
+      final List<Map<dynamic, dynamic>>? serializedResults =
           (await (_channel.invokeListMethod('query', <String, dynamic>{
         'modelName': modelType.modelName(),
         'queryPredicate': where?.serializeAsMap(),
         'queryPagination': pagination?.serializeAsMap(),
         'querySort': sortBy?.map((element) => element.serializeAsMap()).toList()
-      })))!;
-
+      })));

```

Comment:
nit: As the `!` is removed, the outer `()` should be no longer needed

Reply:
ah thanks good catch